test/acceptance/features/steps/openshift.py

```diff
@@ -1,5 +1,6 @@
 import re
 import time
+import base64
```

Comment:
Remove this, we're not using anywhere else, this will likely cause pylint error.